From khali@linux-fr.org Sun Feb  5 14:29:43 2006
Date: Sun, 5 Feb 2006 23:30:26 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: [PATCH 10/11] i2c: Fix i2c-ite name initialization
Message-Id: <20060205233026.58d9fea4.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-ite-name-init.patch

Properly set the name member of the i2c_adapter structure of the ite
i2c adapter driver.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/i2c/busses/i2c-ite.c |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

--- gregkh-2.6.orig/drivers/i2c/busses/i2c-ite.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-ite.c
@@ -200,9 +200,7 @@ static struct i2c_adapter iic_ite_ops = 
 	.owner		= THIS_MODULE,
 	.id		= I2C_HW_I_IIC,
 	.algo_data	= &iic_ite_data,
-	.dev		= {
-		.name	= "ITE IIC adapter",
-	},
+	.name		= "ITE IIC adapter",
 };
 
 /* Called when the module is loaded.  This function starts the
